mirgecom/discretization.py

```diff
@@ -45,18 +45,19 @@ def create_discretization_collection(actx, mesh, order, *, mpi_communicator=None
     """Create and return a grudge DG discretization object."""
     from grudge.dof_desc import DISCR_TAG_BASE, DISCR_TAG_QUAD
     from grudge.eager import EagerDGDiscretization
-    from meshmode.discretization.poly_element import \
-        QuadratureSimplexGroupFactory, \
-        PolynomialRecursiveNodesGroupFactory
+    from meshmode.discretization.poly_element import (
+        QuadratureSimplexGroupFactory,
+        default_simplex_group_factory
```

Comment:
Why is this change needed? it just calls PolynomialRecursiveNodesGroupFactory anyway...
https://github.com/inducer/meshmode/blob/main/meshmode/discretization/poly_element.py#L952

Reply:
This one is more robust in that it will continue to work even if the TPL `recursivenodes` is not installed.   Older installations will not have this, and reproducing an older run will need this more robust version to work.

Reply:
ok, import wrapper in case `recursivenodes` is unavailable.